### DISCUSSION TITLE
Instrumenta for Excel?

### DISCUSSION THREAD

**Discussion by alphapat** [iappyx/Instrumenta#18] [Ideas]
Hey I love you quick access toolbar for PowerPoint and i am using it every day.
It would be cool if you can do something like that for Excel as well :)
Or do you know someone who did it for Excel? 

Would be great :) Thank you

**Comment by iappyx** [comment]
Hi @alphapat, could look into that. Personally I don't use any additional toolbars for Excel. What kind of features would you expect?

**Reply by alphapat** [reply]
I found something similar but i have to add the QAT every time i close excel.
Maybe you can build something similar and better :)

https://github.com/koirand/quick-excel-toolbar

**Comment by iappyx** [comment]
Will consider an Excel version of Instrumenta, but may take some time.

This QAT toolbar already seems quite extensive.
Not sure why this one needs to be added every time you open Excel in your set-up. It seems to use the same basic principles as Instrumenta. Are you saving it into a temporary location?
